packages/manager/src/features/Billing/BillingPanels/PaymentInfoPanel/PayPalChip.tsx

```diff
@@ -101,10 +102,10 @@ export const PayPalChip = (props: Props) => {
 
     return actions.braintree
       .tokenizePayment(data)
-      .then((payload) => onNonce(payload.nonce));
+      .then((payload) => onNonce(payload.nonce, queryClient));
   };
 
-  const onNonce = (nonce: string) => {
+  const onNonce = (nonce: string, queryClient: QueryClient) => {
```

Comment:
```suggestion
  const onNonce = (nonce: string) => {
```

The query client is already in the scope here so we don't need it as a parameter to this function